github/PullRequest.pyi

```diff
@@ -61,8 +61,8 @@ class PullRequest(CompletableGithubObject):
     ) -> PullRequestComment: ...
     def create_review_request(
         self,
-        reviewers: Union[_NotSetType, str] = ...,
-        team_reviewers: Union[_NotSetType, str] = ...,
+        reviewers: Union[_NotSetType, List[str]] = ...,
+        team_reviewers: Union[_NotSetType, List[str]] = ...,
```

Comment:
`delete_review_request` below also needs to be updated.

Reply:
Done